### DISCUSSION TITLE
How to enable Copilot Business?

### DISCUSSION THREAD

**Discussion by yoshinorin** [hexojs/hexo#5390] [General]
@hexojs/core 

Has anyone requested the activation of Copilot Business?
Is it available for free? If possible please tell me how to it. Generally requires a payment of $19/monthly.

![copilot](https://github.com/hexojs/hexo/assets/11273093/22f344f1-38bd-4d4c-b075-aad33ea3350a)
![copilot2](https://github.com/hexojs/hexo/assets/11273093/e8760559-ba58-4064-8ca9-bd08ef45eba0)
![copilot3](https://github.com/hexojs/hexo/assets/11273093/bfe4f48b-8855-47d0-88d9-d38a297ffbc2)

**Comment by uiolee** [comment]
**"Copilot Business"** requires a fee. Not available for free.
https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#pricing-for-github-copilot-business

**"Copilot Individual"** is free for some people.
https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#pricing-for-github-copilot-individual

>A free subscription for GitHub Copilot is available to verified students, teachers, and maintainers of popular open-source repositories on GitHub. If you meet the criteria as an open source maintainer, you will be automatically notified when you visit the GitHub Copilot subscription page

**Reply by yoshinorin** [reply]
Thank you :)
I close this issue and mark as done it on my inbox.
